flang/lib/Optimizer/CodeGen/CodeGen.cpp

```diff
@@ -89,7 +89,7 @@ static constexpr unsigned kAttrAllocatable = CFI_attribute_allocatable;
 // Instantiate static data member of the type converter.
 StringMap<mlir::Type> fir::LLVMTypeConverter::identStructCache;
 
-inline mlir::Type getVoidPtrType(mlir::MLIRContext *context) {
+static mlir::Type getVoidPtrType(mlir::MLIRContext *context) {
```

Comment:
nit: `static inline` to keep the hint? (I think `inline` is ok under the LLVM coding standard, no?)

Reply:
I did not find anything in the coding standard that disallows inline.
Have created a patch upstream. Will include it here once it is merged. https://reviews.llvm.org/D115619